lib/puppet/transaction.rb

```diff
@@ -134,14 +134,14 @@ def evaluate(&block)
     teardown = lambda do
       # Just once per type. No need to punish the user.
       providerless_types.uniq.each do |type|
-        Puppet.err "Could not find a suitable provider for #{type}"
+        Puppet.err _("Could not find a suitable provider for #{type}")
       end
 
       post_evalable_providers.each do |provider|
         begin
           provider.post_resource_eval
         rescue => detail
-          Puppet.log_exception(detail, "post_resource_eval failed for provider #{provider}")
+          Puppet.log_exception(detail, _("post_resource_eval failed for provider #{provider}"))
```

Comment:
@LarissaLane added a note to the doc https://docs.google.com/document/d/1NPa1uZYD8gqlMx7guuZlji6TheMfrQYRCjtJQxYg_10/edit to avoid translating `_` separated terms like this one